urdfenvs/dual_arm/envs/.gitignore

```diff
@@ -0,0 +1 @@
+__pycache__/
```

Comment:
I feel like tests/.pytest_cache should also be added. What do you think?

Reply:
Why?